config/keys.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  mongoURI: 'mongodb://systers:systers2018@ds235461.mlab.com:35461/programs'
```

Comment:
can this also be stored in the environment.prod.ts file?

Reply:
@bethanyr thanks for the review :)
The environment.prod.ts file lives on the client side(Angular). The keys.js file lives on the server side(Node). The client side never makes a call to mongodb://systers:systers2018@ds235461.mlab.com:35461/programs
so I don't think it is necessary to add this mongoURI in the environment.prod.ts file.
Plus this mongoURI is being used for development purposes only. The URI that is used for production is https://systers-github-io.herokuapp.com.